src/resdk/resources/background_task.py

```diff
@@ -0,0 +1,73 @@
+"""Process resource."""
+import logging
+from time import sleep, time
+
+from resdk.exceptions import ResolweServerError
+
+from .base import BaseResource
+
+
+class BackgroundTask(BaseResource):
+    """Background task resource.
+
+    :param resolwe: Resolwe instance
+    :type resolwe: Resolwe object
+    :param model_data: Resource model data
+
+    """
+
+    endpoint = "task"
+
+    READ_ONLY_FIELDS = BaseResource.READ_ONLY_FIELDS + (
+        "pk",
```

Comment:
Didd you add this by mistake or is it not documented below?

Reply:
Actually a bit of both. The serializer in the Resolwe class returned `pk` instead of `id`, which is now fixed.